README.md

```diff
@@ -46,16 +46,11 @@ const {
 
 // configure API endpoint and authentication token
 const exporter = new DynatraceMetricExporter({
```

Comment:
Please also update https://github.com/dynatrace-oss/opentelemetry-metric-js/tree/main/samples to allow running it with no URL and token.